desktop_version/src/GraphicsResources.cpp

```diff
@@ -84,7 +86,6 @@ SDL_Surface* LoadImageSurface(const char* filename)
 
     if (optimizedImage == NULL)
     {
-        VVV_free(data);
```

Comment:
This is not actually a double-free, because `VVV_free` checks for `NULL` first and will set it to `NULL` afterwards. But it's still good to remove this.